### ISSUE TITLE
Use cross-fetch instead of isomorphic-fetch

### ISSUE THREAD

**Issue by Otto-AA** [jeff-zucker/solid-rest#11]
(Opening this issue here, so it doesn't get lost)

Copied from https://github.com/jeff-zucker/solid-file-client/issues/59#issuecomment-510402051:

Ok, after some digging into the the node_modules dependencies, I've found out that ismorphic-fetch is using an outdated version. It specifies "node-fetch": "^1.0.1" (=1.x) in its dependencies, so it sticks with the newest 1.x version of node-fetch. But node-fetch is already at v2.0.6, the blob support was added in v2.0.0 as experimental feature (see their changelog).

TL;DR: isomorphic-fetch uses an outdated version of node-fetch which means it doesn't support blobs

My suggestion for solid-auth-cli is, to use cross-fetch instead. It is basically the same as isomorphic-fetch, but with newer versions. And the last commit there was last week, whereas the last commit in isomorphic-fetch was in 2016.

**Comment by Otto-AA** [comment]
Sorry, this was meant to be in the solid-auth-cli repository. I'll move it there....
